### ISSUE TITLE
release of cohttp-lwt-unix

### ISSUE THREAD

**Issue by hannesm** [mirage/ocaml-cohttp#621]
the latest release thereof in opam-repository is 1.0.2, and conflicts with lwt 4.x.  The current HEAD works fine with lwt 4.x -- it would be great to have a new cohttp-lwt-unix release (see as well https://github.com/andreas/ocaml-graphql-server/issues/93).  thanks!

**Comment by hongchangwu** [comment]
Even the latest release on GitHub (1.1.0) still references the deprecated `lwt_logs`. A new release will be nice.

**Comment by djtango** [comment]
I'm completely new to ocaml - is there an easy workaround to this that someone could ELI5 me?

**Comment by samoht** [comment]
@djtango before the actual release, you can use the dev version by doing:

```
opam pin add cohttp-lwt-unix --dev
```

**Comment by hongchangwu** [comment]
Ping. Can we cut a new release at least on GitHub?

**Comment by mseri** [comment]
I agree. I’ll get one asap

**Comment by hannesm** [comment]
@mseri thanks, this just got merged.
